apps/web/pages/api/basenames/frame/frameResponses.ts

```diff
@@ -0,0 +1,125 @@
+import { getFrameMetadata, getFrameHtmlResponse } from '@coinbase/onchainkit/frame';
```

Comment:
It looks like these responses are all used in only one place--it probably makes sense to define them in the same file as the consuming logic.

Reply:
i went back and forth on this. Ultimately, I like that all of these are located in the same file because it makes it much easier for me to figure out the flow of one frame to the next. 

But if you feel strongly that this is bad practice I can change it.